joblib/numpy_pickle.py

```diff
@@ -28,6 +32,18 @@
 from ._compat import _basestring, PY3_OR_LATER
 from .backports import make_memmap
 
+# Register supported compressors
+register_compressor('zlib', CompressorWrapper(obj=BinaryZlibFile,
+                                              prefix=_ZLIB_PREFIX,
+                                              extension='.z'))
+register_compressor('gzip', CompressorWrapper(obj=BinaryGzipFile,
```

Comment:
Why don't you define the `ZLibCompressorWrapper` and the `GzipCompressorWrapper` in the `compressor.py` file? This would be more consistent and avoid duplicationg these calls here and in the tests

Reply:
Added the ZlibCompressorWrapper and GzipCompressorWrapper classes in compressor.py. And updated the tests.